### ISSUE TITLE
Raw mouse input has double sensitivity with SDL2

### ISSUE THREAD

**Issue by adem4ik** [ValveSoftware/steam-for-linux#3842]
Hi there, we are on the way to Steam release of our game and decided to switch from X11 to SDL2 on Linux, since it is suggested for developers by Valve. But there is a problem appeared with raw mouse input - it has doubled sensitivity. It is very important for our games, since most of them are hardcore players who needs as much precision as possible and only raw mouse input can give it. So we checked a few games from Steam for Linux and found that CS:GO and TF2 have that problem too. Also there is a lot of topics through the internet reporting the same issue:
http://forums.steampowered.com/forums/showthread.php?p=36005482
http://steamcommunity.com/groups/homestream/discussions/1/540736587115642793/
https://steamcommunity.com/app/221410/discussions/0/666825525500565719/
https://www.reddit.com/r/GlobalOffensive/comments/2mo96u/comprehensive_csgo_linux_guide_with_steamdesktop/cm6myxl

So basically it affects on every game which uses SDL2 for raw mouse input. But there is solution for this bug:
http://lists.libsdl.org/pipermail/sdl-libsdl.org/2015-April/000288.html
https://bugzilla.libsdl.org/show_bug.cgi?id=2963

It would be nice if Steam-for-Linux developers could help this bug to get a proper attention. Also it would be great if you can update SDL2 library in your runtime environment if that bug gets fixed someday.

**Comment by kisak-valve** [comment]
Hello @adem4ik, the steam runtime variant was updated a good while ago and if there are other affected games, it should be brought to the game devs attention on a per-game basis.

Closing in favor of https://github.com/ValveSoftware/Source-1-Games/issues/1834 and https://github.com/ValveSoftware/csgo-osx-linux/issues/570.
